macosx/Controller.mm

```diff
@@ -3580,10 +3580,19 @@ - (void)checkAutoImportDirectory
             continue;
         }
 
+        NSDictionary<NSFileAttributeKey, id>* fileAttributes = [NSFileManager.defaultManager attributesOfItemAtPath:fullFile
+                                                                                                              error:nil];
+        if (fileAttributes.fileSize == 0)
+        {
+            // Workaround for Firefox downloads happening in two steps: first time being an empty file
+            [self.fAutoImportedNames removeObject:file];
+            continue;
+        }
+
         auto metainfo = tr_torrent_metainfo{};
         if (!metainfo.parseTorrentFile(fullFile.UTF8String))
         {
-            break;
+            continue;
```

Comment:
This was another hidden bug, I suppose?
Like, when multiple files downloaded at the same time and one of them is broken in any way - everything else was skipped.

Reply:
Yes